dockers/docker-database/database_config.json.j2

```diff
@@ -3,7 +3,7 @@
         "redis":{
             "hostname" : "127.0.0.1",
             "port" : 6379,
-            "unix_socket_path" : "/var/run/redis/redis.sock",
+            "unix_socket_path" : "/var/run/redis{{NAMESPACE_ID}}/redis.sock"
```

Comment:
we need the comma ",", right?  No error when parsing the json file with this PR ?

Reply:
Oh great ! thanks Dong .. I was also debugging a build failure - could be related to this. I did update this j2 file with this attribute "persistence_for_warm_boot" yesterday - missed the comma. Shall update.